### PR TITLE
Doc: Create doc for etcdctl check perf command

### LINKED ISSUE #9072: doc: write doc for perf check of etcdctl
The perf check command needs to be documented.

https://github.com/coreos/etcd/blob/master/etcdctl/ctlv3/command/check.go

@spzala do you want to take this one?

### PR DESCRIPTION
**Pull request by spzala**
Create a doc for etcdctl check commands with perf subcommand.

Fixes #9072

# Contributing guidelines

Please read our [contribution workflow][contributing] before submitting a pull request.

[contributing]: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#contribution-flow

### CODE REVIEW

File: etcdctl/README.md [hunk 1]
```diff
@@ -1440,6 +1440,43 @@ Prints etcd version and API version.
...
+# PASS: Throughput is 150 writes/s
+# PASS: Slowest request took 0.087509s
+# PASS: Stddev is 0.011084s
+# PASS
+./etcdctl check perf --load="l"
+# 60 / 60 Booooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo! 100.00%1m0s
+# FAIL: Throughput too low: 6808 writes/s
+# PASS: Slowest request took 0.228191s
+# PASS: Stddev is 0.033547s
+# FAIL
```
Comment (xiang90) [inline etcdctl/README.md]: probably explain a little bit why this example fails? this is testing a large load on a laptop?

**Comment by xiang90** [comment]
lgtm

File: etcdctl/README.md (hunk 1 above)
Comment (spzala) [reply etcdctl/README.md]: Sure, that's a good idea. Yes, I guess the failure is due to what you mentioned - large workload on a single node cluster on laptop. Thanks!

Comment (xiang90) [reply etcdctl/README.md]: Great! Let us document the reason of the failure, and get this PR merged :)

Comment (spzala) [reply etcdctl/README.md]: :) yup!!

**Comment by spzala** [comment]
retest

**Comment by xiang90** [comment]
@spzala 

filed issue about the test failure:https://github.com/coreos/etcd/issues/9086

**Comment by spzala** [comment]
@xiang90 thanks but I still don't understand why this PR changes making semaphore fail. First time I noticed the TestWatchRestore related failure and thought it could be something just a one time failure for some reason and I guess it does look that way because this time it's a different error than TestWatchRestore. I also run .semaphore.sh on my local VM and that went fine.

**Comment by spzala** [comment]
Last time, 386-integration and amd64-integration both failed but this time it appears that it's only amd64- failing with different error as I mentioned.

**Comment by xiang90** [comment]
@spzala it is not because of this PR i believe. probably a test flakes.

**Comment by spzala** [comment]
@xiang90 thanks!!

**Pull request merged**